packages/dds/tree/src/simple-tree/api/view.ts

```diff
@@ -103,13 +103,7 @@ export class ViewSchema {
 		// and its impossible for an adapter to be correctly implemented if its output type is never
 		// (unless its input is also never).
 		for (const adapter of this.adapters?.tree ?? []) {
-			if (
-				isNeverTree(
-					this.policy,
-					this.storedSchema,
-					this.storedSchema.nodeSchema.get(adapter.output),
-				)
-			) {
+			if (isNeverTree(this.policy, this.schema, this.schema.nodeSchema.get(adapter.output))) {
```

Comment:
(Just a heads up: was reading the docs above this and found a small existing typo. "it's probably a bug it they use never types" should say "if" instead of "it")